src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -266,24 +280,14 @@ def setup(self, builder: Builder):
         super().setup(builder)
         self.clock = builder.time.clock()
         self.time_step = builder.configuration.time.step_size  # in days
-        assert (
-            self.time_step <= 30
-        ), "DecennialCensusObserver requires model specification configuration with time.step_size <= 30"
+        if self.time_step > 30:
```

Comment:
We can remove this error now since `to_observe` now works with longer timesteps.